app/src/main/java/com/todoist_android/ui/auth/LoginFragment.kt

```diff
@@ -42,18 +64,5 @@ class LoginFragment : BaseFragment<AuthenticationViewModel, FragmentLoginBinding
             //TODO: add validation
             viewModel.login(email, password)
         }
-
-        binding.textViewRegister.setOnClickListener {
```

Comment:
navigating to Signup Fragment with clickable link ... we won't need this?

Reply:
Let me pull to get the recent changes